templates/staging/ia-archiver-cronjob.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: ia-import
+spec:
+  schedule: "30 3 * * * "
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: ia-import
+            image: envirodgi/ia-archiver:latest
+            env:
+                - name: WEB_MONITORING_DB_URL
+                  valueFrom:
+                      secretKeyRef:
+                        name: app-secrets
```

Comment:
Was just looking at this again because of your other comment and realized: should this be a reference to the DB service in the same cluster? e.g. how we connect the DB to the differ: https://github.com/edgi-govdata-archiving/web-monitoring-ops/blob/468fc65ff91c5275565e22f2726beb5da29c0750/templates/production/api-deployment.yaml#L64-L65